app/views/accounts/account/_credit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="relative p-4 border border-gray-100 bg-offwhite rounded-xl focus-within:bg-white focus-within:shadow focus-within:opacity-100">
+  <%= f.label :subtype, t('.type') ,class: 'block text-sm font-medium opacity-50 focus-within:opacity-100'  %>
```

Comment:
Nit - to match the surrounding code

```suggestion
  <%= f.label :subtype, t(".type") ,class: "block text-sm font-medium opacity-50 focus-within:opacity-100"  %>
```

Reply:
Good catch! I think we should think about in adding Rubocop asap and detect this kind of errors automatically.